platform/android/java/app/src/com/godot/game/GodotApp.java

```diff
@@ -55,10 +53,4 @@ public class GodotApp extends GodotActivity {
 			}
 		}
 	}
-
-	@Override
-	public void onCreate(Bundle savedInstanceState) {
-		SplashScreen.installSplashScreen(this);
```

Comment:
The logic for splash screen setup should remain as is; splash screen requires additional configurations that the `lib` module can't automatically provide, and so it must be repeated.

Reply:
Updated